influx/influx.go

```diff
@@ -222,7 +231,7 @@ func (cr *ChunkedResponse) Next() ([]int64, []float64, error) {
 	}
 	values := make([]float64, len(fieldValues))
 	for i, fv := range fieldValues {
-		v, err := toFloat64(fv)
+		v, err := convertToFloat64(fv)
```

Comment:
I guess, its better to add additional case into convertToFloat64 function:

https://github.com/VictoriaMetrics/vmctl/blob/master/influx/parser.go#L38

something like: 
```go
func toFloat64(v interface{}) (float64, error) {
	switch i := v.(type) {
	case float64:
		return i, nil
        case json.Number:
                return i.Float64(),nil
```

Reply:
@kotlarz @f41gh7 it should be just a two-line change in `toFloat64` function:
```
 func toFloat64(v interface{}) (float64, error) {
        switch i := v.(type) {
+       case json.Number:
+               return i.Float64()
        case float64:
                return i, nil

```